alfresco-transform-libreoffice/alfresco-transform-libreoffice-boot/src/test/java/org/alfresco/transformer/LibreOfficeTransformationIT.java

```diff
@@ -84,7 +91,13 @@
         testFile(MIMETYPE_OPENDOCUMENT_SPREADSHEET,"ods",null),
         testFile(MIMETYPE_PDF,"pdf",null),
         testFile(MIMETYPE_TSV,"tsv",null),
-        testFile(MIMETYPE_EXCEL,"xls",null)
+        testFile(MIMETYPE_EXCEL,"xls",null),
```

Comment:
It looks like you've added a lot of unnecessary Targets in spreadsheetTargets and docTargets. Looking at what transforms are added to L/O in the merge --> https://github.com/Alfresco/alfresco-transform-core/pull/246/files#diff-ea090e9f38a105fca06b78337718d996L225, they all go to pdf, which is already present in existing targets list, so all those need to be removed and are the reason you are getting many "by-product" unnecessary failing tests. I can implement a quick fix for that.

Reply:
This is done now

Reply:
Yes, as per @KikoUK let's focus on adding specific tests for the supported transforms we've added in https://github.com/Alfresco/alfresco-transform-core/pull/246/. 

Obviously next time we're adding extra supported transforms (ie. Core T-Engine config) then we should add the tests at the same time to avoid lag.